[Jul 23 21:17:08] INFO[15490]: asterisk.test_case:173 __init__: Executing tests/manager/acl-login
[Jul 23 21:17:08] DEBUG[15490]: asterisk.test_conditions:143 register_observer: Registering: <function handle_condition_failure at 0x7fea355897d0>
[Jul 23 21:17:08] DEBUG[15490]: asterisk.test_conditions:143 register_observer: Registering: <function handle_condition_failure at 0x7fea355897d0>
[Jul 23 21:17:08] DEBUG[15490]: asterisk.test_conditions:143 register_observer: Registering: <bound method AMILoginACLTest.handle_condition_failure of <__main__.AMILoginACLTest object at 0x7fea350b6bd0>>
[Jul 23 21:17:08] INFO[15490]: asterisk.test_case:278 create_asterisk: Creating Asterisk instance 1
[Jul 23 21:17:08] INFO[15490]: asterisk.test_case:444 _start_asterisk: Starting Asterisk instance 1
[Jul 23 21:17:08] DEBUG[15490]: asterisk.asterisk:239 connectionMade: Asterisk 127.0.0.1 - connection made
[Jul 23 21:17:09] DEBUG[15490]: asterisk.asterisk:863 cli_exec: Executing ['/usr/sbin/asterisk', '-C', '/tmp/asterisk-testsuite/c18e99a89b256c594723f6622cd93879/run_1/ast1/etc/asterisk/asterisk.conf', '-rx', 'core waitfullybooted'] ...
[Jul 23 21:17:11] DEBUG[15490]: asterisk.asterisk:179 __cli_output_callback: Asterisk CLI 127.0.0.1 exited 0
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 0 - test1 / 127.0.0.1
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea350e10e0>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 1 - test1 / 127.0.0.2
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea350e1638>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 2 - test1 / 127.0.0.3
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea350e1950>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 3 - test1 / 127.0.0.4
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea350e1c68>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 4 - test3 / 127.0.0.1
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea350e1f80>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 5 - test3 / 127.0.0.2
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c072d8>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 6 - test3 / 127.0.0.3
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c075f0>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 7 - test3 / 127.0.0.4
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c07908>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 8 - test2 / 127.0.0.1
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c07c20>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 9 - test2 / 127.0.0.2
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c07f38>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 10 - test2 / 127.0.0.3
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c22290>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 11 - test2 / 127.0.0.4
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c225a8>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 12 - test4 / 127.0.0.1
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c228c0>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 13 - test4 / 127.0.0.2
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c22bd8>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 14 - test4 / 127.0.0.3
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c22ef0>
[Jul 23 21:17:11] INFO[15490]: __main__:137 ami_factory_test_function: Creating AMI 15 - test4 / 127.0.0.4
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fea34c29248>
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test1', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884215112-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test1', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884103008-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test1', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643889191896-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test1', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643889159128-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test3', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643889159920-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test3', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884241336-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test3', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884242416-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test3', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884242632-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test2', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884242848-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test2', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884243064-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test2', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884243280-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test2', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884243712-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test4', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884244216-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test4', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884244720-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test4', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884273960-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:184 connectionMade: Connection Made
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'login', 'username': 'test4', 'secret': 'mysecret', 'actionid': '35cc2ba0add2-140643884274464-1'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884215112-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884215112-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884215112-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 0: login authenticated
[Jul 23 21:17:11] INFO[15490]: __main__:96 ami_connect: AMI 0: Login successful and expected.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884215112-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643889159920-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643889159920-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643889159920-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 4: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 4: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643889159920-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884241336-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884241336-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884241336-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 5: login authenticated
[Jul 23 21:17:11] INFO[15490]: __main__:96 ami_connect: AMI 5: Login successful and expected.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884241336-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884242416-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884242416-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884242416-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 6: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 6: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884242416-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884242632-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884242632-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884242632-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 7: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 7: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884242632-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884242848-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884242848-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884242848-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 8: login authenticated
[Jul 23 21:17:11] INFO[15490]: __main__:96 ami_connect: AMI 8: Login successful and expected.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884242848-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884243064-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884243064-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884243064-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 9: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 9: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884243064-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884243280-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884243280-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884243280-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 10: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 10: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884243280-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884243712-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884243712-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884243712-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 11: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 11: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884243712-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Asterisk Call Manager/5.0.0'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884244216-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884244216-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884244216-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 12: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 12: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884244216-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884244720-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884244720-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884244720-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 13: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 13: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884244720-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884215112-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884215112-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643889159920-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643889159920-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884241336-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884241336-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884242416-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884242416-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884242632-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884242632-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884242848-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884242848-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884243064-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884243064-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884243280-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884243280-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884243712-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884243712-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884273960-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884273960-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884273960-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 14: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 14: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884273960-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Success'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884274464-1'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication accepted'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884274464-1'}
[Jul 23 21:17:11] INFO[15490]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140643884274464-1'}
[Jul 23 21:17:11] INFO[15490]: __main__:84 ami_connect: AMI 15: login authenticated
[Jul 23 21:17:11] ERROR[15490]: __main__:92 ami_connect: AMI 15: Login was allowed and it shouldn't have been.
[Jul 23 21:17:11] DEBUG[15490]: AMI:361 sendMessage: MSG OUT: {'action': 'logoff', 'actionid': '35cc2ba0add2-140643884274464-2'}
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea350e1170> will retry in 3 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea350e10e0>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c07050> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea350e1f80>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c07368> will retry in 3 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c072d8>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c07680> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c075f0>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c07998> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c07908>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c07cb0> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c07c20>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c07fc8> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c07f38>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c22320> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c22290>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c22638> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c225a8>
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884244216-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884244216-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884244720-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884244720-2'}
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c22950> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c228c0>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c22c68> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c22bd8>
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884274464-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884274464-2'}
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Goodbye'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884273960-2'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Thanks for all the fish.'
[Jul 23 21:17:11] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:11] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:11] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Thanks for all the fish.', u'response': u'Goodbye', u'actionid': u'35cc2ba0add2-140643884273960-2'}
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c292d8> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c29248>
[Jul 23 21:17:11] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea34c22f80> will retry in 2 seconds
[Jul 23 21:17:11] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:11] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea34c22ef0>
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Error'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643884103008-1'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication failed'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:12] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:12] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication failed', u'response': u'Error', u'actionid': u'35cc2ba0add2-140643884103008-1'}
[Jul 23 21:17:12] INFO[15490]: AMI:210 onFailure: Login Call Failure: Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 174, in lineReceived
    
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 269, in dispatchIncoming
    
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 393, in callback
    self._startRunCallbacks(result)
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 501, in _startRunCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 587, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 401, in errorUnlessResponse
    
starpy.error.AMICommandFailure: {u'message': u'Authentication failed', u'response': u'Error', u'actionid': u'35cc2ba0add2-140643884103008-1'}

[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Error'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643889191896-1'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication failed'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:12] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:12] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication failed', u'response': u'Error', u'actionid': u'35cc2ba0add2-140643889191896-1'}
[Jul 23 21:17:12] INFO[15490]: AMI:210 onFailure: Login Call Failure: Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 174, in lineReceived
    
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 269, in dispatchIncoming
    
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 393, in callback
    self._startRunCallbacks(result)
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 501, in _startRunCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 587, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 401, in errorUnlessResponse
    
starpy.error.AMICommandFailure: {u'message': u'Authentication failed', u'response': u'Error', u'actionid': u'35cc2ba0add2-140643889191896-1'}

[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Response: Error'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'ActionID: 35cc2ba0add2-140643889159128-1'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: 'Message: Authentication failed'
[Jul 23 21:17:12] DEBUG[15490]: AMI:171 lineReceived: Line In: ''
[Jul 23 21:17:12] DEBUG[15490]: AMI:233 dispatchIncoming: Dispatch Incoming
[Jul 23 21:17:12] DEBUG[15490]: AMI:263 dispatchIncoming: Incoming Message: {u'message': u'Authentication failed', u'response': u'Error', u'actionid': u'35cc2ba0add2-140643889159128-1'}
[Jul 23 21:17:12] INFO[15490]: AMI:210 onFailure: Login Call Failure: Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 174, in lineReceived
    
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 269, in dispatchIncoming
    
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 393, in callback
    self._startRunCallbacks(result)
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 501, in _startRunCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 587, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/lib/python2.7/site-packages/starpy-1.0.2-py2.7.egg/starpy/manager.py", line 401, in errorUnlessResponse
    
starpy.error.AMICommandFailure: {u'message': u'Authentication failed', u'response': u'Error', u'actionid': u'35cc2ba0add2-140643889159128-1'}

[Jul 23 21:17:12] ERROR[15490]: __main__:157 evaluate_success: An AMI login was allowed which was not expected.
[Jul 23 21:17:12] INFO[15490]: asterisk.test_case:487 __stop_instances: Stopping Asterisk instance 1
[Jul 23 21:17:12] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea350e16c8> will retry in 3 seconds
[Jul 23 21:17:12] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea350e1638>
[Jul 23 21:17:12] DEBUG[15490]: asterisk.asterisk:561 __send_stop_gracefully: sending stop gracefully
[Jul 23 21:17:12] DEBUG[15490]: asterisk.asterisk:863 cli_exec: Executing ['/usr/sbin/asterisk', '-C', '/tmp/asterisk-testsuite/c18e99a89b256c594723f6622cd93879/run_1/ast1/etc/asterisk/asterisk.conf', '-rx', 'core stop gracefully'] ...
[Jul 23 21:17:12] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea350e19e0> will retry in 2 seconds
[Jul 23 21:17:12] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea350e1950>
[Jul 23 21:17:12] INFO[15490]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fea350e1cf8> will retry in 2 seconds
[Jul 23 21:17:12] INFO[15490]: AMI:1139 clientConnectionLost: None
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fea350e1c68>
[Jul 23 21:17:12] INFO[15490]: asterisk.test_case:508 __stop_reactor: Stopping Reactor
[Jul 23 21:17:12] INFO[15490]: twisted:154 publishToNewObserver: Main loop terminated.
